### PR TITLE
Fixed clang-format supports v3.8

### PR DESCRIPTION
**Pull request by Alexey-N-Chernyshov**
I removed some more keys.

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -55,8 +53,6 @@ IncludeCategories:
 IndentCaseLabels: true
 IndentWidth:     4
 IndentWrappedFunctionNames: false
-JavaScriptQuotes: Leave
-JavaScriptWrapImports: true
```
Comment (afiskon) [inline .clang-format:59]: ОК, we don't need those

File: .clang-format [hunk 2]
```diff
@@ -33,8 +33,6 @@ BreakBeforeBinaryOperators: None
 BreakBeforeBraces: Attach
 BreakBeforeTernaryOperators: true
 BreakConstructorInitializersBeforeComma: true
-BreakAfterJavaFieldAnnotations: false
```
Comment (afiskon) [inline .clang-format:36]: OK

File: .clang-format [hunk 3]
```diff
@@ -33,8 +33,6 @@ BreakBeforeBinaryOperators: None
...
-BreakStringLiterals: true
```
Comment (afiskon) [inline .clang-format:37]: OK

File: .clang-format [hunk 4]
```diff
@@ -75,7 +71,6 @@ PointerAlignment: Left
 ReflowComments:  false
 SortIncludes:    true
 SpaceAfterCStyleCast: false
-SpaceAfterTemplateKeyword: false
```
Comment (afiskon) [inline .clang-format:78]: This one causes changes in defer.h shown below, but I don't mind.

File: src/HttpServer.cpp [hunk 5]
```diff
@@ -152,7 +152,8 @@ HttpRequestHandler HttpWorker::_chooseHandler(HttpRequest& req) {
             int mvector[32];
             const size_t mvectorItems = sizeof(mvector) / sizeof(mvector[0]);
             int rc = pcre_exec(listItem->regexp, nullptr, query, query_len, 0, 0, mvector, mvectorItems);
-            if(rc >= 0) { /* match! */
+            if(rc >= 0) {
+                /* match! */
```
Comment (afiskon) [inline src/HttpServer.cpp:156]: Are you sure this is a right formatting after running `make format` ? I have Clang 4.0.0 and it doesn't change this. What version of Clang are you using?

File: src/HttpServer.cpp [hunk 6]
```diff
@@ -335,10 +336,10 @@ void HttpServer::accept(const std::atomic_bool& terminate_flag) {
...
 
     /*
-     * We need to increase workersCounter in the parent process to prevent race
-     * condition. Otherwise it's possible that parent process will terminate
-     * before child process increments a counter.
-     */
+   * We need to increase workersCounter in the parent process to prevent race
+   * condition. Otherwise it's possible that parent process will terminate
+   * before child process increments a counter.
+   */
```
Comment (afiskon) [inline src/HttpServer.cpp:342]: Same question as above. That's just not very pretty. Perhaps you could rewrite it somehow to make your version of clang-format happy?

File: src/Socket.cpp [hunk 7]
```diff
@@ -79,7 +79,8 @@ size_t Socket::readLine(char* buff, size_t buffsize) {
 
         if((currsize >= 2) && (buff[currsize - 1] == '\n') &&
            ((buff[currsize - 2] == '\r') || (buff[currsize - 2] == '\n' /* see [1] */))) {
-            /* [1] - it's for debugging purpose only, in case developer decided to use `nc` */
+            /* [1] - it's for debugging purpose only, in case developer decided to use 
+             * `nc` */
```
Comment (afiskon) [inline src/Socket.cpp:83]: Meh... OK, I can live with that.

**Review by afiskon — CHANGES_REQUESTED** [review]

**Comment by Alexey-N-Chernyshov** [comment]
My fault, comments were modified by the default style. I'll better create a new pull request to not show wrong commits in the history.

**Pull request closed without merging**